### ISSUE TITLE
NMM 0.62.16 - Refreshing Online Profiles list causes a trace log error

### ISSUE THREAD

**Issue by KunoMochi** [Nexus-Mods/NMM-Test-Builds#103]
**Issue:**
When refreshing the list of Online Profiles, a trace log error occurs. Still occurs after reinstalling NMM (and removing all contents during uninstallation). This occurs right when the 4th step of the refresh starts

```
Tracing an Unhandled Exception:
Exception: 
Message: 
    Input string was not in a correct format.
```

[TraceLog20160524154905.txt](https://github.com/Nexus-Mods/NMM-Test-Builds/files/280378/TraceLog20160524154905.txt)

**Steps to reproduce:**
1. Launch NMM
2. Go to Profiles tab
3. Click on the "Refresh Online Profiles list" icon

![nmm0-62-16_refreshonlineprofiles](https://cloud.githubusercontent.com/assets/7691651/15517831/e5f78c1a-21c7-11e6-87b8-8f062bcd91f1.png)

**Comment by KunoMochi** [comment]
**Additional note:**
Uncertain if this is related but I am also unable to upload a new profile to the server. Profile upload progress is displayed but upon completing, it does not bring up the profile page to edit when I had chosen "Yes" nor does the Online Profiles list get refreshed.

**Comment by DuskDweller** [comment]
Thanks Kuno, we found out where's the issue, but we're trying to understand what is causing it, I'll have a look at your online profiles.

**Comment by AraFox** [comment]
[TraceLog20160528104405.txt](https://github.com/Nexus-Mods/NMM-Test-Builds/files/287783/TraceLog20160528104405.txt)

Confirmed here too.

**Comment by DuskDweller** [comment]
Fixed on 0.62.17 :)
